api/.php_cs.dist

```diff
@@ -0,0 +1,37 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->exclude(['bin', 'config', 'docker', 'helm', 'var', 'vendor'])
+    ->in(__DIR__)
+;
+
+return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@DoctrineAnnotation' => true,
+        '@Symfony' => true,
+        'array_syntax' => ['syntax' => 'short'],
```

Comment:
Better to keep the nesting level consistent by expanding the array, even if it's only one option for now.